packages/documentation/src/pages/architecture/site-structure/bundles-and-code-organization.mdx

```diff
@@ -1,4 +1,41 @@
-# Bundle and Code Organization
+# Bundles and Code Organization
+
+## Application Bundling with Webpack
```

Comment:
I think we should go with sentence case for these headings. I've been doing that elsewhere and it's the general VA.gov content guideline.